assets/src/dashboard/components/card-grid-item/index.js

```diff
@@ -46,6 +46,6 @@ const CardGridItem = ({ children, ...rest }) => {
 };
 
 CardGridItem.propTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
this is throwing a warning right now since the CardGridItem isn't fleshed out yet.